docs/docs/fault-domain-awareness.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Fault Domain Awareness
+---
+
+# Fault Domain Awareness
+
+## Overview
+
+A fault domain is a set of nodes that share similar failure and latency characteristics. Two nodes in the same fault domain are both affected by failure events within the domain. Placing nodes in more than one fault domain reduces the risk that a failure will affect both nodes.
+
+Marathon supports fault domain awareness, so that you can mitigate the risk of fault domain failure for high availability and configure your apps and pods to take advantage of region awareness to allow for increased capacity when needed.
```

Comment:
Sentence feels run on; split it up?

Reply:
we should state the specific type of fault domain we are referring to.
Marathon previously supported node and rack fault domains... Marathon now supports region.   
Although now as I'm writing it... I'm not sure we REALLY do.   We do support workload bursting to other regions which is part of the equation.